doc/rest.md

```diff
@@ -195,6 +195,11 @@ Content-Type: `application/json`
 JSON object with following structure:
 - **name**: String. Layername used for identification within given [workspace](models.md#workspace). It can be also used for identifying layer within WMS and WFS endpoints.
 - **uuid**: String. UUID of the layer.
+- **layman_metadata**
+  - **publication_status**: String. Can be one of these values:
```

Comment:
Mention `publication_status` also in POST Layers line 99 for watching global state